### PR TITLE
docs(widget): sounds, tokens, images, banners

### PR DESCRIPTION
**Pull request by shoom3301**
# Description

Docs for the features:

- [feat(widget): use custom sounds in widget](https://github.com/cowprotocol/cowswap/pull/4162)
- [feat(widget): use custom tokens as virtual list](https://github.com/cowprotocol/cowswap/pull/4164)
- [feat(widget): customize images](https://github.com/cowprotocol/cowswap/pull/4159)
- [feat(widget): option to hide safe banner](https://github.com/cowprotocol/cowswap/pull/4175)

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @shoom3301 , great!
However, some issues/questions:
1. I noticed that not only MP3 sounds can be supported, but WAV and OGG files as well. WDYT about adding this info as well? 
ogg and wav


2. I'd add here qoutes at least. Or maybe formatted 'null' as a code
as  a code
3. For images, WDYT about adding supported formats and sizes? :) 
fromats
If not in the table, but maybe inside the 'Customization' section

Thank you!

File: docs/cow-protocol/tutorials/widget/widget.md [hunk 1]
```diff
@@ -131,24 +131,28 @@ createCowSwapWidget(
...
+| `disableToastMessages` | `boolean`              | false                  | CoW Swap displays a pop-up notification when certain events occur, for example: an order has been filled. You may want to handle these events yourself and disable the display of notifications in the widget, to do this you need to enable this option. See [Events handling](#events-handling) section for more details. |
+| `standaloneMode`       | `boolean`              | false                  | When this option is enabled, the widget will use its own Ethereum provider and the user will be able to connect a wallet from within the widget.                                                                                                                                                                            |
+| `hideLogo`             | `boolean`              | false                  | Option to hide the logo in the widget.                                                                                                                                                                                                                                                                                      |
```
Comment (anxolin) [inline docs/cow-protocol/tutorials/widget/widget.md]: do we still use this setting? do we need it?

File: docs/cow-protocol/tutorials/widget/widget.md [hunk 2]
```diff
@@ -131,24 +131,28 @@ createCowSwapWidget(
...
+| `banners`              | `CowSwapWidgetBanners` | ---                    | With this option you can control the display of some banners in the CoW Swap interface.                                                                                                                                                                                                                                     |
+| `tokenLists`           | `string[]`             | ---                    | CowSwap uses the [token lists standard](https://tokenlists.org), which allows flexible and decentralized management of assets for trading.                                                                                                                                                                                  |
+| `customTokens`         | `TokenInfo[]`          | ---                    | In the case when you want to use tokens that are not included in the default token lists, you can add them programmatically by simply passing an array of ERC-20 tokens info.                                                                                                                                               |
```
Comment (anxolin) [inline docs/cow-protocol/tutorials/widget/widget.md]: Probably nice to mention also `tokenLists` option as its related

File: docs/cow-protocol/tutorials/widget/widget.md [hunk 3]
```diff
@@ -131,24 +131,28 @@ createCowSwapWidget(
...
+| `images`               | `CowSwapWidgetImages`  | ---                    | To make your widget unique, you can replace some of the images. For example, a stub image when there are no limit orders.                                                                                                                                                                                                   |
+| `banners`              | `CowSwapWidgetBanners` | ---                    | With this option you can control the display of some banners in the CoW Swap interface.                                                                                                                                                                                                                                     |
+| `tokenLists`           | `string[]`             | ---                    | CowSwap uses the [token lists standard](https://tokenlists.org), which allows flexible and decentralized management of assets for trading.                                                                                                                                                                                  |
```
Comment (anxolin) [inline docs/cow-protocol/tutorials/widget/widget.md]: We could maybe add `See also [customTokens] option`  as they are related

File: docs/cow-protocol/tutorials/widget/widget.md [hunk 4]
```diff
@@ -131,24 +131,28 @@ createCowSwapWidget(
...
+| `tradeAssets`          | `TradeAssets`          | Same as in swap.cow.fi | An object containing information about the selling and buying assets. Example: `{ asset: 'WBTC', amount: 12 }` or `{ asset: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48' }`.                                                                                                                                                |
+| `theme`                | `CowSwapTheme`         | 'light'                | The theme of the widget (`'dark'` for dark theme or `'light'` for light theme). It is also possible to set your own colors (`CowSwapWidgetPalette`). See [Custom theme](#custom-theme) section for more details.                                                                                                            |
+| `sounds`               | `CowSwapWidgetSounds`  | ---                    | The famous MOOO! sound can be changed to something else, just provide a link to the media file (mp3). You can also specify a value of null to disable the sound.                                                                                                                                                            |
```
Comment (anxolin) [inline docs/cow-protocol/tutorials/widget/widget.md]: ❤️

**Review by anxolin — APPROVED** [review]

File: docs/cow-protocol/tutorials/widget/widget.md (hunk 2 above)
Comment (shoom3301) [reply docs/cow-protocol/tutorials/widget/widget.md:149]: It's just a few lines above :)

File: docs/cow-protocol/tutorials/widget/widget.md [hunk 5]
```diff
@@ -187,6 +191,72 @@ Try it yourself: https://widget.cow.fi.
...
+
+> How to create and deploy your own token list you can find in the [guide](https://hackmd.io/@mastercow/creating-custom-token-lists).
+
+```typescript
+import { createCowSwapWidget, CowSwapWidgetParams } from '@cowprotocol/widget-lib'
+
+const container = document.getElementById('cowswap-widget')
+
+const params: CowSwapWidgetParams = {
+    tokenLists: ['https://gateway.ipfs.io/ipns/tokens.uniswap.org']
```
Comment (alfetopito) [inline docs/cow-protocol/tutorials/widget/widget.md]: Can we suggest by default a token list that's our own instead of uniswap's?

**Review by alfetopito — APPROVED** [review]

File: docs/cow-protocol/tutorials/widget/widget.md (hunk 1 above)
Comment (shoom3301) [reply docs/cow-protocol/tutorials/widget/widget.md]: Nice catch!
Removed it

File: docs/cow-protocol/tutorials/widget/widget.md (hunk 5 above)
Comment (shoom3301) [reply docs/cow-protocol/tutorials/widget/widget.md]: Set https://files.cow.fi/tokens/CowSwap.json

File: docs/cow-protocol/tutorials/widget/widget.md (hunk 3 above)
Comment (shoom3301) [reply docs/cow-protocol/tutorials/widget/widget.md]: Fixed

File: docs/cow-protocol/tutorials/widget/widget.md (hunk 2 above)
Comment (shoom3301) [reply docs/cow-protocol/tutorials/widget/widget.md]: Fixed

**Comment by shoom3301** [comment]
@elena-zh thank you!

1. Added "mp3 or any other audio format supported by your browser"
2. Fixed
3. I think this is not necessary, integrators should know best which formats and sizes they want to use

**Review by elena-zh — APPROVED** [review]

**Pull request merged**